### PR TITLE
create index boilerplate

### PR DESCRIPTION
**Pull request by Coderuff919**
Add index boilerplate

### CODE REVIEW

File: main/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+
```
Comment (mikesz88) [inline main/index.html:8]: delete extra space

File: main/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="global.css">
+
```
Comment (mikesz88) [inline main/index.html:10]: delete extra space

File: main/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+
```
Comment (mikesz88) [inline main/index.html:14]: delete extra space

File: main/index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+
+</head>
+
```
Comment (mikesz88) [inline main/index.html:18]: delete extra space

File: main/index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+<body>
+
+    <div class="navbar"></div>
+
```
Comment (mikesz88) [inline main/index.html:22]: delete extra space

File: main/index.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="hero"></section>
+
```
Comment (mikesz88) [inline main/index.html:24]: delete extra space

File: main/index.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="amazing-services"></section>
+
```
Comment (mikesz88) [inline main/index.html:26]: delete extra space

File: main/index.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="page-creator-experience"></section>
+
```
Comment (mikesz88) [inline main/index.html:28]: delete extra space

File: main/index.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="pricing-plan"></section>
+
```
Comment (mikesz88) [inline main/index.html:30]: delete extra space

File: main/index.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="awesome-clients"></section>
+
```
Comment (mikesz88) [inline main/index.html:32]: delete extra space

File: main/index.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="thoughts-and-experiments"></section>
+
```
Comment (mikesz88) [inline main/index.html:34]: delete extra space

File: main/index.html [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="busin-tech"></section>
+
```
Comment (mikesz88) [inline main/index.html:36]: delete extra space

File: main/index.html [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+    <section class="footer"></section>
+
+</body>
+
```
Comment (mikesz88) [inline main/index.html:40]: delete extra space

**Review by mikesz88 — CHANGES_REQUESTED** [review]
Let's take out the unnecessary spaces, and we will be good! Let me know when you are done!

File: main/index.html (hunk 2 above)
Comment (Coderuff919) [reply main/index.html:10]: OK, let me revise and resubmit

**Review by mikesz88 — APPROVED** [review]
LGTM!

**Pull request merged**